### DISCUSSION TITLE
Feature Request: Tracking License Usage

### DISCUSSION THREAD

**Discussion by homerbanzon** [Kong/insomnia#5060] [Ideas]
Would like a way to track actual Insomnia license usage.  This is to make sure that team members assigned a license are actually putting them to use.  If the license manager had some kind of portal to see who the licenses are assigned to and how many minutes of usage, plus last login/usage date, it would help us maintain good user stats.

**Comment by bmirkic** [comment]
And, if I may add, please make that report exportable in either csv, xlsx, XML or similar.
